### ISSUE TITLE
upgrade busybox to a new version to avoid security vulnerabilities

### ISSUE THREAD

**Issue by jtcheng** [GoogleContainerTools/distroless#1014]
From file `busybox_archives.bzl`
For `busybox_amd64`, `busybox_s390x` and `busybox_arm` platforms we still use an old version `1.31.0` busybox.
An image scan tool found below issues:
CVE-2018-1000500
CVE-2021-42382
CVE-2021-42384
CVE-2021-42378
CVE-2021-42379
CVE-2021-42380
CVE-2021-42381
CVE-2021-42385
CVE-2021-42386
CVE-2021-42376
CVE-2021-42374

**Comment by loosebazooka** [comment]
It doesn't look like busybox has published multiarch binaries yet, so we need to wait on that.

**Comment by jtcheng** [comment]
@loosebazooka 
may we can use the per-arch binaries (such as [busybox.tar.xz](https://github.com/docker-library/busybox/blob/dist-amd64/stable/musl/busybox.tar.xz)) as done for `busybox_arm64`?

**Comment by loosebazooka** [comment]
I mean, sure, PRs are welcome. Debug images just generally aren't on the critical path, so it's low priority on our side.
